plugins/woocommerce/src/Database/Migrations/CustomOrderTable/CLIRunner.php

```diff
@@ -137,7 +138,7 @@ public function count_unmigrated( $args = array(), $assoc_args = array() ) : int
 	}
 
 	/**
-	 * Migrate order data to the custom orders table.
+	 * Sync order data between the custom orders table and wp_posts table.
```

Comment:
Totally optional, this is just a musing:

```suggestion
	 * Sync order data between the custom order tables and the core WordPress post tables.
```

(On the basis that `wp_` is not always the prefix, and because there are multiple tables on both sides.)